packages/scaffolder/src/command.ts

```diff
@@ -0,0 +1,77 @@
+import chalk from 'chalk';
+
+import { getFeatureStore, initializeFeatureStore } from './features/store';
+import { initializeLogger } from './logger';
+import { configToGenerator, promptUserForFeature } from './features';
+import commandArguments from './arguments';
+import { initializeConfigurationStore } from './configuration';
+import handleError from './error';
+
+class ScaffolderCommand {
+  private readonly arguments: typeof commandArguments & { _unknown?: string[] };
+
+  private logger: ReturnType<typeof initializeLogger>;
+
+  public constructor() {
+    this.arguments = commandArguments;
+
+    const {
+      debug = false,
+      root = process.cwd(),
```

Comment:
Since process.cwd is set as the fallback below if root is falsy, I'd recommend defaulting it to false here as well and only specifying the fallback in one place